### PR TITLE
MINOR: Switched to new Signing Server endpoint and more

### PR DESCRIPTION
**Pull request by kohlerpop1**
- Switched to new Signing Server enpoint
- Added detection for age restricted lives that cannot be connected to 
 -- (potential to bypass if session_id is set, still experimenting)
- Removed SignServerResponse & SignServerMapper as now its a direct request & response of bytes
- Optimized exception messages to be cleared on failure response
- Added LiveClientSettings#apiKey to use an API key if one is obtained

### CODE REVIEW

File: Client/src/main/java/io/github/jwdeveloper/tiktok/http/mappers/LiveDataMapper.java [hunk 1]
```diff
@@ -64,6 +64,10 @@ public LiveData.Response map(String json) {
                 default -> LiveData.LiveStatus.HostNotFound;
             };
             response.setLiveStatus(statusValue);
+        } else if (data.has("prompts") && jsonObject.has("status_code") &&
+            data.get("prompts").getAsString().isEmpty() && jsonObject.get("status_code").isJsonPrimitive()) {
+            // 4003110 is age restriction code
```
Comment (jwdeveloper) [inline Client/src/main/java/io/github/jwdeveloper/tiktok/http/mappers/LiveDataMapper.java:69]: 4003110 is Magic number, Please put it to some kind of Constant variable

File: Client/src/main/java/io/github/jwdeveloper/tiktok/TikTokLiveHttpClient.java [hunk 2]
```diff
@@ -210,49 +188,29 @@ public LiveConnectionData.Response fetchLiveConnectionData(LiveConnectionData.Re
...
 
-    SignServerResponse getSignedUrl(String roomId) {
-        var urlToSign = httpFactory
-                .client(TikTokLiveHttpClient.TIKTOK_URL_WEBCAST + "im/fetch")
-                .withParam("room_id", roomId)
-                .build()
-                .toUrl();
+    HttpResponse<byte[]> getStarterPayload(String room_id) {
+        HttpClientBuilder builder = httpFactory.client(TIKTOK_SIGN_API)
+            .withParam("client", "ttlive-java")
```
Comment (jwdeveloper) [inline Client/src/main/java/io/github/jwdeveloper/tiktok/TikTokLiveHttpClient.java:193]: I think we should keep client parameter for metricts. Or it was removed from sign server?

**Review by jwdeveloper — COMMENTED** [review]
Review, please resolve comments

File: Client/src/main/java/io/github/jwdeveloper/tiktok/TikTokLiveHttpClient.java (hunk 2 above)
Comment (kohlerpop1) [reply Client/src/main/java/io/github/jwdeveloper/tiktok/TikTokLiveHttpClient.java:193]: Correct, we no longer need the url. Only the current parameters and room_id passing directly to the signing server.

File: Client/src/main/java/io/github/jwdeveloper/tiktok/http/mappers/LiveDataMapper.java (hunk 1 above)
Comment (kohlerpop1) [reply Client/src/main/java/io/github/jwdeveloper/tiktok/http/mappers/LiveDataMapper.java]: Converted to constant!

**Pull request merged**